jubatus/core/classifier/nearest_neighbor_classifier.cpp

```diff
@@ -256,6 +260,29 @@ void nearest_neighbor_classifier::unlearn_id(const std::string& id) {
   nearest_neighbor_engine_->get_table()->delete_row(id);
 }
 
+void nearest_neighbor_classifier::unlearn_label(const std::string& label) {
```

Comment:
The method name and its function mismatch.  How about `decrement_label_counter`?